README.md

```diff
@@ -90,6 +90,29 @@ module.exports.options = {
 }
 ```
 
+#### fastify version discovery
+
+If fastify has been installed local to plugin.js that copy will be used instead of the global copy included with fastify-cli.
```

Comment:
I understand what is being said here because I know the details behind it. But this would be really confusing as a newcomer. Maybe:

"If Fastify is installed as a project dependency, `npm install --save fastify`, then `fastify-cli` will use that version of Fastify when running the server. Otherwise, `fastify-cli` will use the version of Fastify included within `fastify-cli`."